utils/epinow2/functions.py

```diff
@@ -9,6 +10,29 @@
 )
 
 
+def extract_user_args() -> dict:
+    """Extracts user-provided arguments from environment variables or uses default if none provided."""
+    state = os.environ.get("state", "all")
+    disease = os.environ.get("disease", "all")
+    report_date = os.environ.get("report_date", date.today())
+    min_reference_date, max_reference_date = get_reference_date_range(report_date)
+    reference_dates = os.environ.get(
+        "reference_date", [min_reference_date, max_reference_date]
+    )
+    data_source = os.environ.get("data_source", "nssp")
+    data_path = os.environ.get("data_path", "gold/")
+    data_container = os.environ.get("data_container", None)
+    return {
+        "state": state,
+        "disease": disease,
+        "report_date": report_date,
+        "reference_dates": reference_dates,
+        "data_source": data_source,
+        "data_path": data_path,
+        "data_container": data_container,
+    }
+
+
 def generate_timestamp() -> int:
     """Generates a timestamp of the current time using UTC timezone."""
     return int(datetime.timestamp(datetime.now(timezone.utc)))
```

Comment:
This is mostly personal preference, but I would rather the timestamp was formatted string as opposed to an int. @zsusswein, what do you think?

Reply:
+1

Reply:
will do!